xpti/include/xpti/xpti_data_types.h

```diff
@@ -386,20 +386,46 @@ enum class trace_point_type_t : uint16_t {
   /// Used to notify that memory has been released.
   mem_release_end = XPTI_TRACE_POINT_END(19),
   /// Used to notify that offload buffer will be created
-  offload_alloc_construct = XPTI_TRACE_POINT_BEGIN(20),
+  offload_alloc_buffer_construct = XPTI_TRACE_POINT_BEGIN(20),
   /// Used to notify about association between user and internal
   /// handle of the offload buffer
-  offload_alloc_associate = XPTI_TRACE_POINT_BEGIN(21),
+  offload_alloc_buffer_associate = XPTI_TRACE_POINT_BEGIN(21),
   /// Used to notify that offload buffer will be destructed
-  offload_alloc_destruct = XPTI_TRACE_POINT_BEGIN(22),
+  offload_alloc_buffer_destruct = XPTI_TRACE_POINT_BEGIN(22),
   /// Used to notify about releasing internal handle for offload buffer
-  offload_alloc_release = XPTI_TRACE_POINT_BEGIN(23),
+  offload_alloc_buffer_release = XPTI_TRACE_POINT_BEGIN(23),
   /// Used to notify about creation accessor for offload buffer
   offload_alloc_accessor = XPTI_TRACE_POINT_BEGIN(24),
   /// User to notify when a queue has been created
   queue_create = XPTI_TRACE_POINT_BEGIN(25),
   /// User to notify when a queue has been destroyed
   queue_destroy = XPTI_TRACE_POINT_END(25),
+  /// Used to notify that offload sampled image will be created
+  offload_alloc_sampled_image_construct = XPTI_TRACE_POINT_BEGIN(26),
+  /// Used to notify about association between user and internal
+  /// handle of the offload sampled image
+  offload_alloc_sampled_image_associate = XPTI_TRACE_POINT_BEGIN(27),
+  /// Used to notify that offload sampled image will be destructed
+  offload_alloc_sampled_image_destruct = XPTI_TRACE_POINT_BEGIN(28),
+  /// Used to notify about releasing internal handle for offload sampled image
+  offload_alloc_sampled_image_release = XPTI_TRACE_POINT_BEGIN(29),
```

Comment:
Similarly, *_associate/*_release should be XPTI_TRACE_POINT_BEGIN(X)/_END(X) if they are semantically efer to the begin and end of the association.
`offload_alloc_sampled_image_release = XPTI_TRACE_POINT_BEGIN(27)`